common/src/test/java/org/opensearch/sql/common/grok/BasicTest.java

```diff
@@ -113,22 +113,22 @@ public void test006_testLoadPatternFromFileIso_8859_1() throws IOException, Grok
     try (FileOutputStream fis = new FileOutputStream(temp);
          BufferedWriter bw = new BufferedWriter(
              new OutputStreamWriter(fis, StandardCharsets.ISO_8859_1))) {
-      bw.write("TEST §");
+      bw.write("TEST #");
     }
 
     GrokCompiler compiler = GrokCompiler.newInstance();
     compiler.register(new FileInputStream(temp), StandardCharsets.ISO_8859_1);
     Grok grok = compiler.compile("%{TEST}");
-    assertEquals("(?<name0>§)", grok.getNamedRegex());
+    assertEquals("(?<name0>#)", grok.getNamedRegex());
   }
 
   @Test
   public void test007_testLoadPatternFromReader() throws IOException, GrokException {
-    Reader reader = new StringReader("TEST €");
+    Reader reader = new StringReader("TEST $");
```

Comment:
This is changing the nature of the test. The test specifically uses non-ascii characters.

It's best to use unicode escape [sequence](https://www.javatpoint.com/java-escape-characters) here.

Reply:
Thanks, fixed in 55182bf31.